Fixes #6679 - Updates Out-String (#6687)
MicrosoftDocs/PowerShell-Docs: reference/5.1/Microsoft.PowerShell.Utility/Out-String.md

## DESCRIPTION

The `Out-String` cmdlet converts input objects into strings. By default, `Out-String`
accumulates the strings and returns them as a single string, but you can use the **Stream**
parameter to direct `Out-String` to return one line at a time or create and array of strings. This
cmdlet lets you search and manipulate string output as you would in traditional shells when object
manipulation is less convenient.

## EXAMPLES

### Example 1: Get the current culture and convert the data to strings

This example gets the regional settings for the current user and converts the object data to

> strings.

Reviewer: Add an output block for this example.

> If you omit the **Stream** parameter, the command displays all the aliases because `Select-String`
> finds the text **gcm** in the single string that `Out-String` returns.

### Example 3: Use the Width parameter to prevent truncation.

> While most output from `Out-String` is wrapped to the next line if it's longer than the specified
> width, there are some scenarios where the output will be truncated due to the formatter cmdlet
> `Out-String` uses for formatting. You can prevent truncation by using the **Width** parameter.

Reviewer: ```suggestion
While most output from `Out-String` is wrapped to the next line, there are scenarios where the
output is truncated by the formatting system before being passed to `Out-String`. You can avoid
truncation using the **Width** parameter.
```
